### PR TITLE
button card text-utility components added

### PR DESCRIPTION
**Pull request by Shinde-Sandesh**
image

image

image

### CODE REVIEW

**Review by iHarryD — APPROVED** [review]
Everything looks good. Can merge.

**Comment by Shinde-Sandesh** [comment]
> Everything looks good. Can merge.

Thank you for review

File: Components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
+.VariantContainer
+{
+      margin-top: 25px;
```
Comment (j24m) [inline Components/button/button.css:3]: Suggestion: You can use rem when the value is more than 10px everywhere :)

File: Components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+{
+      text-decoration: none;
+}
+  
+.btn i
+{
+      color: var(--white-color);
+}
+  
+.primary-btn
```
Comment (j24m) [inline Components/button/button.css:35]: follow a consistent pattern everywhere, for example :- use btn-primary, btn-secondary, btn-warning. You have followed this pattern in btn-success below, that is correct.

File: Components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+       height: 2rem;
+       margin-bottom: 1rem;
+       align-items: center;
+       margin-left: 0.35rem;
+       color: var(--white-color);
+       background-color: var(--blue-color);
+       font-size: 1rem;
+       border: none;
+       border-radius: 10px;
+       /* transition: .3s all ease-in-out; */
```
Comment (j24m) [inline Components/card/card.css:71]: if there is no use of this code you can remove it ?

**Review by j24m — APPROVED** [review]
Hey, your code is good, except for a few changes here and there, just a bit more consistency with indentations will do the magic, but your class names are really good, and UI is dope 🔥
Good work :)

**Comment by j24m** [comment]
> image
> 
> image
> 
> image

In the card image you can maybe provide a little bit of padding, because the button is sticking to the edge.

File: Components/card/card.css (hunk 3 above)
Comment (Shinde-Sandesh) [reply Components/card/card.css:71]: Thank you for reviewing. I'll make changes as per suggestion

**Pull request merged**